### DISCUSSION TITLE
Approved for GitHub Student Developer Pack but Haven’t Received Access

### DISCUSSION THREAD

**Discussion by sengyik-x1** [community/community#144816] [GitHub Education]
### Hi everyone,

I recently applied for the GitHub Student Developer Pack, and my application status shows as approved. However, I haven't received access to the benefits or any confirmation email about the pack.

Here’s some additional context:

Application Date: 14-11-2024
Approval Date: 14-11-2024
GitHub Account Email: sengyik77@gmail.com
Verification Document: student card 

I’ve already checked:

My registered email inbox and spam folder.
The GitHub Education Benefits page to see if access is active.
I’d appreciate any advice or guidance on resolving this. Has anyone else faced a similar issue?

Thank you in advance for your help!

**Comment by davevad93** [comment]
Hi @sengyik-x1 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @sengyik-x1 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
